README.md

```diff
@@ -2,11 +2,10 @@
 [![CI Status](https://github.com/se-edu/addressbook-level3/workflows/Java%20CI/badge.svg)](https://github.com/AY2122S1-CS2103T-W16-3/tp/actions)
```

Comment:
Next time it might be better to do documentation related work on another branch and add it in as a separate PR instead 😅